marlowe-playground-client/src/Examples/JS/Contracts.purs

```diff
@@ -235,18 +235,18 @@ couponBondGuaranteed =
                                                                                                     Pay(investor, Party(guarantor), ada, 1010n, Close)
                                                                                                 )
                                                                                             )
-                                                                                            ], 20n, Close)
+                                                                                            ], 25n, Close)
                                                                                     )
                                                                                 )
                                                                             )
-                                                                        ], 15n, Close)
+                                                                        ], 20n, Close)
                                                                     )
                                                                 )
                                                             )
-                                                        ], 10n, Close)
+                                                        ], 15n, Close)
                                                     )
                                                 )
-                                            ], 5n, Close)
+                                            ], 10n, Close)
```

Comment:
we should really run tests that check our contracts don't have any linting warnings, I presume these were warnings?

Reply:
Yes, I was thinking the same. [SCP-1981](https://jira.iohk.io/browse/SCP-1981)